Added examples for finder document scanner

matthewrindel opened the pull request.

scan-mayank commented on ScanbotSDKExamples/NET/ReadyToUseUI.Droid/MainActivity.cs:
```diff
@@ -178,6 +178,19 @@ private void OnButtonClick(object sender, EventArgs e)
+                configuration.SetCameraPreviewMode(CameraPreviewMode.FitIn);
+                configuration.SetIgnoreBadAspectRatio(true);
+                configuration.SetTextHintOK("Don't move.\nCapturing document...");
+                //configuration.SetFinderLineColor(Color.Blue);
```
we can remove these comments

matthewrindel replied:
These are left as an example, but I have moved the configuration lines under the comment.

scan-mayank commented on ScanbotSDKExamples/NET/ReadyToUseUI.iOS/Controller/MainViewController.cs:
Minor most point: 
There are a few unnecessary new lines, we can remove it.

scan-mayank commented:
I got a few build-time errors, as the newer beta with FinderDocumentScanner is missing.

scan-mayank reviewed (commented):
The code looks good. 
But there are a few new lines, comments which we can remove.

matthewrindel commented:
> The code looks good. But there are a few new lines, comments which we can remove.

Thanks @scan-mayank, I've updated the packages to the latest beta and you can give it a final look. 

In terms of comments, I have updated the examples for what I have changed, but also the comments are often meant to be illustrative and to enable a developer to uncomment to see a different kind of change. So I've done my best to make it clearer that they are demonstrative, but this won't apply to any existing comments (which we can always look at for future changes).
